application/view/common/asset.phtml

```diff
@@ -0,0 +1,65 @@
+    <div class="attachments collapsible" data-template="<?php echo $escape($attachmentRowTemplate); ?>">
+      <?php if ($block !== ''): ?>
+        <?php foreach($attachments as $attachment): ?>
+          <div class="attachment">
+            <span class="sortable-handle"></span>
+            <?php 
+                $asset = $attachment['asset'];
+                $assetId = $asset->id();
+                $page = (array_key_exists('page', $attachment)) ? $attachment['page'] : '';
+                $pageId = ($page !== '') ? $page->id() : '';
+                $altLinkTitle = ($page !== '') ? $attachment['alt_link_title'] : '';
+                $caption = $attachment['caption'];
+            ?>
+            <?php $thumbnail = '<div class="thumbnail"><img src="' . $asset->assetUrl() . '" alt="' . $asset->name() . '"></div>'; ?>
+            <div class="asset-title"><?php echo $thumbnail; ?><?php echo $asset->name(); ?></div>
+            <ul class="actions">
+                <li><?php echo $this->hyperlink('', '#', ['class' => 'asset-options-configure o-icon-configure button', 'title' => $translate('Open asset options')]); ?></li>
+                <li class="delete"><?php echo $this->hyperlink('', '#', ['class' => 'o-icon-delete button', 'title' => $translate('Delete attachment')]); ?></li>
+                <li class="undo"><?php echo $this->hyperlink('', '#', ['class' => 'o-icon-undo button', 'title' => $translate('Restore attachment')]); ?></li>
+            </ul>
+            <input type="hidden" class="asset-page-id asset-option" name="o:block[__blockIndex__][o:data][__attachmentIndex__][page]" value="<?php echo $pageId; ?>" data-page-title="<?php echo ($page !== '') ? $page->title() : ''; ?>" data-page-url="<?php echo ($page !== '') ? $page->siteUrl() : ''; ?>">
+            <input type="hidden" class="asset" name="o:block[__blockIndex__][o:data][__attachmentIndex__][id]" value="<?php echo $assetId; ?>">
+            <input type="hidden" class="alternative-link-title asset-option" name="o:block[__blockIndex__][o:data][__attachmentIndex__][alt_link_title]" value="<?php echo $altLinkTitle; ?>">
+            <input type="hidden" class="asset-caption asset-option" name="o:block[__blockIndex__][o:data][__attachmentIndex__][caption]" value="<?php echo $caption; ?>">
+          </div>
+        <?php endforeach; ?>
+        <?php endif; ?>
+        <button type="button" class="add-asset-attachment" data-sidebar-content-url="<?php echo $this->escapeHtml($this->url('admin/default', ['controller' => 'asset', 'action' => 'sidebar-select'])); ?>"><?php echo $translate('Add asset'); ?></button>
+    </div>
+</div>
+<a class="collapse" href="#" aria-label="<?php echo $translate('collapse'); ?>">
+  <h4>Options</h4>
+</a>
+<div class="collapsible">
+  <div class="field">
+    <div class="field-meta">
+      <label for="o:block[__blockIndex__][o:data][className]"><?php echo $translate('Class'); ?></label>
```

Comment:
This _should_ be the same `for` problem as above but isn't seemingly for some reason at runtime... is there a script that is copying the input name to its id? Odd...